bin/lessc

```diff
@@ -481,8 +482,17 @@ function processPluginQueue() {
                 break;
             case 'sm':
             case 'strict-math':
+                console.warning('Strict math is deprecated. Use --math');
```

Comment:
This warning could specify `--math=parens`, maybe.

Reply:
Fair, yes. Will change it.